app/public/map.js

```diff
@@ -93,8 +93,20 @@ var nazarethNuggets = (function () { // eslint-disable-line
 
   addIconsToMap(nuggets)
 
+  var locationSelector = document.querySelector('.location-select-display')
   var centerButton = document.querySelector('.center-button')
   centerButton.addEventListener('click', function (e) {
     mymap.locate({setView: true})
   })
+
+  var tickButton = document.querySelector('.location-select-tick')
+  tickButton.addEventListener('click', function (e) {
+    locationSelector.classList.toggle('visible')
+    // toggle form tab not visible -- future goooal
+  })
+
+  var infoCrossBtn = document.querySelector('.location-select-cross')
```

Comment:
maybe we should change the variable name to something else cause infoCrossBtn i might this it is the cross for the info tab